### DISCUSSION TITLE
Data Export Task Adding Numbers to File Name

### DISCUSSION THREAD

**Discussion by argeincharge** [dbeaver/dbeaver#20875] [Q&A] [answered]
### Description

Hello. I am currently using DBeaver Enterprise 23.1.0 and I have been running into an issue where, when I schedule a data export task, the second time the task runs, it no longer overwrites the file but instead adds a long string of numbers to the end of the file path. I have another DBeaver instance on a virtual machine running the same export, but that overwrites the file without adding the string of numbers. The other version is 22.2.0 so did something happen in the update to 23.1.0 that no longer overwrites files? I have compared both task settings and they are identical. I rely on this task overwriting as it fires off additional automations I have running and ensuring it is the most recent data is critical
![image](https://github.com/dbeaver/dbeaver/assets/141948769/aba48a68-0a8a-4357-891a-2d42ef39010f)

### DBeaver Version

Enterprise 23.1.0

### Database and driver

Postgres + Redshift

**Comment by E1izabeth** [comment, marked as answer]
Please check your data transfer settings here 
![image](https://github.com/dbeaver/dbeaver/assets/28875055/b089ba9d-7fd2-42ff-a8f4-3fc107fc0a9e)

We don't override files anymore by default as it may cause the data loss for users who don't expect it.

**Comment by argeincharge** [comment]
That has solved my issue, thank you. Wondering if for a future release, if this is the path forward, could that drop down be auto expanded so people are aware of the settings?
Thank you!

**Comment by E1izabeth** [comment]
We want it to be collapsed by default as not so many users use it, but those settings grab a lot of space on the page
